velox/expression/DynamicLibraryLoader.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/expression/DynamicLibraryLoader.h"
+#include <dlfcn.h>
+#include "velox/common/base/Exceptions.h"
+
+namespace facebook::velox::exec {
+
+static constexpr const char* kSymbolName = "registry";
```

Comment:
I wonder if we should call it something like velox_registry to sort of namespace it ?